spec/classes/archive/commmon_spec.rb

```diff
@@ -15,15 +33,13 @@
         git
         cmake
         gcc-c++
+        docker-compose-plugin
       ].each do |p|
         it { is_expected.to contain_package(p) }
       end
 
-      %w[
-        docker-compose
-        cryptography
-      ].each do |p|
-        it { is_expected.to contain_python__pip(p) }
+      if facts[:os]['release']['major'] == '7' && (facts[:os]['architecture'] == 'x86_64')
+        it { is_expected.to contain_python__pip('cryptography') }
```

Comment:
As noted on the puppet code, I believe this pip install can be removed.

Reply:
removed.